browser/resources/settings/brave_overrides/settings_menu.ts

```diff
@@ -97,7 +98,7 @@ RegisterStyleOverride(
 
       @media (prefers-color-scheme: dark) {
         :host {
-          --settings-nav-item-color: #F4F4F4 !important;
+          --settings-nav-item-color: #f4f4f4 !important;
```

Comment:
Why are we using lower case letters here? this does not change anything right?

Reply:
bhaiya mostly i think this happened because of formattor. i will revert it .